tests/unit/common/core/test_middleware.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from pytest_mock import MockerFixture
+
+from common.core import middleware as middleware_module
```

Comment:
Not sure if the aliased import makes sense here, can be just

```suggestion
from common.core.middleware import APIResponseVersionHeaderMiddleware
```

Reply:
Reasons:
1. By importing the module, it is easier to `mocker.patch.object(module, 'member')` rather than `mocker.patch('a.usually.long.path.to.member')`.
2. The name `middleware` sounds nicer as an instance of a Middleware class within tests, hence the alias adding `_module`.

What do you think?

Reply:
Sounds good to me.